lab_4_retrieval_w_clustering/main.py

```diff
@@ -81,6 +103,19 @@ def _calculate_bm25(self, tokenized_document: list[str]) -> Vector:
         Returns:
             Vector: BM25 vector for document.
         """
+        if not tokenized_document or not isinstance(tokenized_document, list) \
+            or not all(isinstance(token, str) for token in tokenized_document):
+            raise ValueError("Bad input")
+
+        vector = [0.0 for _ in self._vocabulary]
```

Comment:
try to create a list from one item and then multiply it by the length of vocabulary